lib/utils/__tests__/azure_storage.test.ts

```diff
@@ -0,0 +1,64 @@
+import { BlobResult, getBlobUrl, upsertBlobFromText } from "../azure_storage";
+
+jest.mock("azure-storage");
+
+import * as azureStorage from "azure-storage";
+
+const aBlobResult: BlobResult = {
+  blobType: "",
+  container: "",
+  contentLength: "",
+  etag: "",
+  lastModified: "",
+  name: "",
+  requestId: ""
+};
+const aConnectionString = "credentials=credentials";
+const aContainerName = "container-name";
+const anAttachmentName = "attachment-name";
+const aRandomText = "random-text";
+
+const aBlobService = new azureStorage.BlobService(aConnectionString);
+
+// function flushPromises<T>(): Promise<T> {
```

Comment:
the commented out function can be removed

Reply:
ok